### ISSUE TITLE
ocr command line tool errors

### ISSUE THREAD

**Issue by antran89** [Azure/azcv#1]
Hi @gjwgit 

Nice to meet you in MSFT-Grab AI workshop. In Grab, we still use azure cmd tool, and find it very useful. Recently, there might be some changes in API, the `ml ocr azcv` cmd does not work anymore with the newest update. Previously, it does not has this error. Could you please take a look?

```
root@jupyter-an-2etran:~/traffic-sign-detection-ocr# ml ocr azcv MULTISHOT_0001_000000.jpg
Traceback (most recent call last):
  File "ocr.py", line 86, in <module>
    rawHttpResponse = client.batch_read_file_in_stream(fstream, mode, custom_headers, raw)
  File "/opt/conda/lib/python3.6/site-packages/azure/cognitiveservices/vision/computervision/operations/_computer_vision_client_operations.py", line 1687, in batch_read_file_in_stream
    header_parameters.update(custom_headers)
ValueError: dictionary update sequence element #0 has length 1; 2 is required
Traceback (most recent call last):
  File "/opt/conda/bin/ml", line 11, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.6/site-packages/mlhub/__init__.py", line 197, in main
    args.func(args)
  File "/opt/conda/lib/python3.6/site-packages/mlhub/commands.py", line 936, in dispatch
    errors = errors.decode("utf-8")
AttributeError: 'NoneType' object has no attribute 'decode'
```

**Comment by antran89** [comment]
Hi @gjwgit 

I figured out the error. It seems that Azure OCR does not need to have `mode` variable of handwritten documents anymore. 

Removing the `mode` variable makes the code working like a charm.
```
rawHttpResponse = client.batch_read_file_in_stream(fstream, custom_headers, raw) 
```

**Comment by gjwgit** [comment]
Thanks @antran89 for posting the issue. The problem was the PyPI package azure.cognitiveservices.vision.computervision was updated from 0.3.0 to 0.4.0 with a non-backward compatible change (removal of the mode argument) as noted in https://pypi.org/project/azure-cognitiveservices-vision-computervision/. Fixed with mlhub model version 2.3.2.
